File: CS/WpfApplication2/ItemToBitmapConverter.cs
using DevExpress.Utils.Svg;
using DevExpress.Xpf.Core.Native;
using System;
using System.Globalization;
using System.IO;
using System.Windows;
using System.Windows.Data;
using System.Windows.Markup;

namespace WpfApplication2 {
    public class ItemToBitmapConverter : MarkupExtension, IValueConverter {
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) {
            string path = @"..\..\Images\" + value.ToString() + ".svg";
            var svgStream = File.OpenRead(path);
            if (svgStream == null)
                return null;
            var svgImage = SvgLoader.LoadFromStream(svgStream);
            var size = new Size(svgImage.Width * ScreenHelper.ScaleX, svgImage.Height * ScreenHelper.ScaleX);
            return WpfSvgRenderer.CreateImageSource(svgImage, size, null, null, true);
        }
        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) {
            throw new NotImplementedException();
        }
        public override object ProvideValue(IServiceProvider serviceProvider) {
            return this;
        }
    }
}

File: VB/WpfApplication2/ItemToBitmapConverter.vb
Imports DevExpress.Utils.Svg
Imports DevExpress.Xpf.Core.Native
Imports System
Imports System.Globalization
Imports System.IO
Imports System.Windows
Imports System.Windows.Data
Imports System.Windows.Markup

Namespace WpfApplication2
	Public Class ItemToBitmapConverter
		Inherits MarkupExtension
		Implements IValueConverter

		Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.Convert
			Dim path As String = "..\..\Images\" & value.ToString() & ".svg"
			Dim svgStream = File.OpenRead(path)
			If svgStream Is Nothing Then
				Return Nothing
			End If
			Dim svgImage = SvgLoader.LoadFromStream(svgStream)
			Dim size = New Size(svgImage.Width * ScreenHelper.ScaleX, svgImage.Height * ScreenHelper.ScaleX)
			Return WpfSvgRenderer.CreateImageSource(svgImage, size, Nothing, Nothing, True)
		End Function
		Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.ConvertBack
			Throw New NotImplementedException()
		End Function
		Public Overrides Function ProvideValue(ByVal serviceProvider As IServiceProvider) As Object
			Return Me
		End Function
	End Class
End Namespace